package.json

```diff
@@ -34,21 +36,23 @@
     "vuex": "^3.0.1"
   },
   "devDependencies": {
-    "@babel/core": "^7.1.0",
     "@babel/preset-env": "^7.3.1",
     "@vue/cli-plugin-babel": "^3.3.0",
     "@vue/cli-service": "^3.3.0",
     "@vue/test-utils": "^1.0.0-beta.29",
+    "babel-core": "^7.0.0-bridge.0",
```

Comment:
@mjhea0 why drop back to the bridge release rather than using 7.1+?

Reply:
Not all of the Babel dependencies are on version 7 yet. There's a sub-dependency somewhere that still depends on 6. That's why the bridge is needed. https://github.com/babel/babel-bridge